src/rgw/rgw_user.cc

```diff
@@ -270,10 +270,10 @@ int rgw_get_user_info_from_index(RGWRados *store, string& key, rgw_bucket& bucke
 int rgw_get_user_info_by_uid(RGWRados *store,
                              const rgw_user& uid,
                              RGWUserInfo& info,
-                             RGWObjVersionTracker *objv_tracker,
-                             real_time *pmtime,
-                             rgw_cache_entry_info *cache_info,
-                             map<string, bufferlist> *pattrs)
+                             RGWObjVersionTracker * const objv_tracker,
+                             real_time * const pmtime,
```

Comment:
To the point of the change, I think you may be going overboard with the constness.  I don't have a principled objection to it though, so I don't object for now to any of these.  We can always relax it.  Casey noted here, let's not add const to non-pointer/reference arguments.